README.md

```diff
@@ -1,9 +1,11 @@
 [![4 GPU Integration Test](https://github.com/pytorch/torchtitan/actions/workflows/integration_test_4gpu.yaml/badge.svg?branch=main)](https://github.com/pytorch/torchtitan/actions/workflows/integration_test_4gpu.yaml?query=branch%3Amain)
 [![8 GPU Integration Test](https://github.com/pytorch/torchtitan/actions/workflows/integration_test_8gpu.yaml/badge.svg?branch=main)](https://github.com/pytorch/torchtitan/actions/workflows/integration_test_8gpu.yaml?query=branch%3Amain)
 
+[![arXiv](https://img.shields.io/badge/arXiv-2410.06511-b31b1b.svg?style=plastic)](https://arxiv.org/abs/2410.06511)
```

Comment:
The badge looks interesting. I personally prefer putting this badge right after the section title "Our torchtitan paper on arXiv". (e.g. the red coloring here looks like suggesting some test failure?)

Reply:
makes sense re: possible mis-interpret red as something failing.
I've moved to just under the section title.